composer.json

```diff
@@ -38,6 +38,7 @@
         "paragonie/random_compat": "^2.0.18",
         "php": ">=7.4",
         "symfony/dotenv": "^4.3 || 5.4",
+        "symfony/yaml": "^5.4",
```

Comment:
It's good that you took same component as Tiki: https://gitlab.com/tikiwiki/tiki/-/blob/master/vendor_bundled/composer.json

Reply:
Done